libpod/networking_linux.go

```diff
@@ -127,13 +127,13 @@ type slirp4netnsCmd struct {
 	Args    slirp4netnsCmdArg `json:"arguments"`
 }
 
-func checkSlirpFlags(path string) (bool, bool, error) {
+func checkSlirpFlags(path string) (bool, bool, bool, error) {
```

Comment:
Nit, it would be nice if you added names for these variables from a documentation pint of view.
checkSlirpFlags(path string) (supportsDisableLoopback, supportsMTU, supportsSandbox bool, err error)